static/js/constants.js

```diff
@@ -88,8 +88,8 @@ export const FA_PENDING_STATUSES = [
 ];
 export const FA_APPROVED_STATUSES = [FA_STATUS_APPROVED, FA_STATUS_AUTO_APPROVED];
 
-export const TOAST_SUCCESS = 'success';
-export const TOAST_FAILURE = 'failure';
+export const TOAST_SUCCESS = 'done';
+export const TOAST_FAILURE = 'error';
```

Comment:
Correct. The only thing that the old code was doing with these constants was comparing them, like this:

```js
if (toastMessage.icon === TOAST_FAILURE) {
  icon = <Icon name="error" key="icon "/>;
} else if (toastMessage.icon === TOAST_SUCCESS) {
  icon = <Icon name="done" key="icon" />;
}
```

So I figure, might as well use `done` and `error` directly.